wasmdome/src/main.rs

```diff
@@ -103,6 +139,127 @@ NKEYs are sensitive and should be treated as secrets.
         Err("Did not obtain valid arena credentials".into())
     }
 }
+
+fn check_schedule(nc: nats::Connection) -> Result<(), Box<dyn Error>> {
+    let res = nc.request_timeout(
+        "wasmdome.public.arena.schedule",
+        "",
+        std::time::Duration::from_millis(500),
+    );
+
+    if let Err(e) = res {
+        println!("Error requesting schedule from the lattice: {}", e);
+        return Ok(());
+    };
+
+    let data = &res.unwrap().data;
+    let matches: Vec<StoredMatch> = serde_json::from_slice(data)?;
+    let mut table = Table::new();
+    if matches.len() > 0 {
+        table.add_row(row![
+            "Match Id",
+            "Match Start",
+            "Board Size (H ⨯ W)",
+            "Max Actors",
+            "Max Turns",
+            "Actions per turn"
```

Comment:
See previous comment... I don't think we need to let developers play with the action points per turn option.